src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -51,6 +51,7 @@
     private String message;
     private String url;
     private Long timeTaken;
+    private static UserType userType;
```

Comment:
No, this must not be done. `UserType` is specific for each instance of `ActivityLogEntry` and thus must not be `static`.